python-sdk/tiedie/api/data_receiver_client.py

```diff
@@ -29,7 +29,7 @@ def __init__(self,
         self.host = host
         self.port = port
         self.authenticator = authenticator
-        self.mqtt_client = mqtt.Client(
+        self.mqtt_client = mqtt.Client(mqtt.CallbackAPIVersion.VERSION2,
```

Comment:
I need to test if this continues to work.

Reply:
Backed off to VERSION1 as it really is a breaking change, opened issue to get us to VERSION2